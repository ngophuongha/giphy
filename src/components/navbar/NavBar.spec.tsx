import { render } from "../../../test";
import { NavBar } from "./index";

describe("Styled Navbar", () => {
  it("should match snapshot", () => {
    const { container } = render(<NavBar />);
    expect(container).toMatchSnapshot();
  });
});
