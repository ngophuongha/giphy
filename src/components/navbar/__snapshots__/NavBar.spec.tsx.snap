// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Navbar should match snapshot 1`] = `
.c0 {
  color: #000000 !important;
}

.c1 {
  color: #000000 !important;
}

<div>
  <nav
    class="border-bottom p-3 navbar navbar-expand-lg navbar-light"
  >
    <div
      class="container"
    >
      <a
        class="c0 navbar-brand"
        href="#home"
      >
        <img
          alt=""
          class="d-inline-block align-top"
          height="30"
          src="/giphy-logo-1.svg"
          width="30"
        />
         
        GIPHY
      </a>
      <button
        aria-controls="responsive-navbar-nav"
        aria-label="Toggle navigation"
        class="navbar-toggler collapsed"
        type="button"
      >
        <span
          class="navbar-toggler-icon"
        />
      </button>
      <div
        class="navbar-collapse collapse"
        id="responsive-navbar-nav"
      >
        <div
          class="me-auto navbar-nav"
        >
          <a
            class="c1 nav-link"
            data-rr-ui-event-key="#"
            href="#"
            role="button"
            tabindex="0"
          >
            Reactions
          </a>
          <a
            class="c1 nav-link"
            data-rr-ui-event-key="#"
            href="#"
            role="button"
            tabindex="0"
          >
            Entertainment
          </a>
        </div>
        <div
          class="align-items-center navbar-nav"
        >
          <div
            style="position: relative; display: inline-block; text-align: left; opacity: 1; direction: ltr; border-radius: 12.5px; transition: opacity 0.25s; user-select: none;"
          >
            <div
              class="react-switch-bg"
              style="height: 25px; width: 56px; margin: 0px; position: relative; background: rgb(68, 68, 68); border-radius: 12.5px; cursor: pointer; transition: background 0.25s;"
            >
              <div
                style="height: 25px; width: 33px; position: relative; opacity: 1; pointer-events: none; transition: opacity 0.25s;"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-sun fa-w-16 h-100 w-100 p-1"
                  color="#ffa701"
                  data-icon="sun"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M256 160c-52.9 0-96 43.1-96 96s43.1 96 96 96 96-43.1 96-96-43.1-96-96-96zm246.4 80.5l-94.7-47.3 33.5-100.4c4.5-13.6-8.4-26.5-21.9-21.9l-100.4 33.5-47.4-94.8c-6.4-12.8-24.6-12.8-31 0l-47.3 94.7L92.7 70.8c-13.6-4.5-26.5 8.4-21.9 21.9l33.5 100.4-94.7 47.4c-12.8 6.4-12.8 24.6 0 31l94.7 47.3-33.5 100.5c-4.5 13.6 8.4 26.5 21.9 21.9l100.4-33.5 47.3 94.7c6.4 12.8 24.6 12.8 31 0l47.3-94.7 100.4 33.5c13.6 4.5 26.5-8.4 21.9-21.9l-33.5-100.4 94.7-47.3c13-6.5 13-24.7.2-31.1zm-155.9 106c-49.9 49.9-131.1 49.9-181 0-49.9-49.9-49.9-131.1 0-181 49.9-49.9 131.1-49.9 181 0 49.9 49.9 49.9 131.1 0 181z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              <div
                style="height: 25px; width: 33px; position: absolute; opacity: 0; right: 0px; top: 0px; pointer-events: none; transition: opacity 0.25s;"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-moon fa-w-16 h-100 w-100 p-1"
                  color="#ffa701"
                  data-icon="moon"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M283.211 512c78.962 0 151.079-35.925 198.857-94.792 7.068-8.708-.639-21.43-11.562-19.35-124.203 23.654-238.262-71.576-238.262-196.954 0-72.222 38.662-138.635 101.498-174.394 9.686-5.512 7.25-20.197-3.756-22.23A258.156 258.156 0 0 0 283.211 0c-141.309 0-256 114.511-256 256 0 141.309 114.511 256 256 256z"
                    fill="currentColor"
                  />
                </svg>
              </div>
            </div>
            <div
              class="react-switch-handle"
              style="height: 23px; width: 23px; background: rgb(255, 255, 255); display: inline-block; cursor: pointer; border-radius: 50%; position: absolute; transform: translateX(32px); top: 1px; outline: 0; border: 0px; transition: background-color 0.25s, transform 0.25s, box-shadow 0.15s;"
            />
            <input
              aria-checked="true"
              checked=""
              role="switch"
              style="border: 0px; height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; width: 1px;"
              type="checkbox"
            />
          </div>
          <a
            class="c1 nav-link"
            data-rr-ui-event-key="2"
            href="#"
            role="button"
            tabindex="0"
          >
            Sign in
          </a>
        </div>
      </div>
    </div>
  </nav>
</div>
`;
